README.rst

```diff
@@ -317,6 +318,34 @@ Get
     print(response)
     # <class pybutton.Response id: customer-1234, ...>
 
+Transactions
```

Comment:
please add description for how this differs from accounts#transactions.

Reply:
Added, not sure if I used the right convention in the description, though.